disnake/webhook/async_.py

```diff
@@ -1193,7 +1193,7 @@ def from_url(
             A partial webhook is just a webhook object with an ID and a token.
         """
         m = re.search(
-            r"discord(?:app)?.com/api/webhooks/(?P<id>[0-9]{17,20})/(?P<token>[A-Za-z0-9\.\-\_]{60,68})",
+            r"discord(?:app)?.com/api/webhooks/(?P<id>[0-9]{17,19})/(?P<token>[A-Za-z0-9\.\-\_]{60,68})",
```

Comment:
Do we know what the token length depends on? I just looked at a few tokens (including one that's more than 3 years old), they all had 68 chars which is already the upper limit for this regex :thinking:

Reply:
I don't, but I don't think that is documented.